src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/services/EnvelopeFinaliserServiceTest.java

```diff
@@ -164,6 +176,8 @@ public void finaliseEnvelope_should_not_update_other_envelopes_with_ocr_data_and
         assertThat(unaffectedEnvelope.get().getStatus()).isEqualTo(Status.NOTIFICATION_SENT);
         assertThat(unaffectedEnvelope.get().getScannableItems())
             .allMatch(item -> item.getOcrData() != null && item.getOcrValidationWarnings() != null);
+        assertThat(unaffectedEnvelope.get().getCcdId()).isNull();
+        assertThat(unaffectedEnvelope.get().getEnvelopeCcdAction()).isNull();
```

Comment:
I wonder if this whole test is just a duplicate of the test above. Maybe we should just remove it?

Reply:
very similar, one has warnings. I will validate in one of them existing ccd ref has not change.